index.html

```diff
@@ -1,5 +1,9 @@
 <html>
+    <titlE>poc 2</titlE>
```

Comment:
improve the title description and use capital

Reply:
I am following the convention

Reply:
please do it

Reply:
done